src/main/kotlin/probation/workspace.kt

```diff
@@ -11,8 +11,14 @@ fun probationWorkspace(): Workspace {
   workspace.model.enterprise = Enterprise("Probation in HM Prison and Probation Service")
 
   CloudPlatform.defineDeploymentNodes(workspace.model)
-  Delius(workspace.model)
+
+  val modelItems = listOf(ProbationPractitioners, EPF, Delius, IM, NID, EQuiP, CRCSystem, Reporting)
```

Comment:
Should we name this `systems` for consistency with the prisons script?

Reply:
I would love to, but `ProbationPractitioners` define only people, and I fear it would confuse readers. I like the freedom to group people/systems together in these classes. However, it may not be guaranteed they define system(s).